### ISSUE TITLE
French Translations

### ISSUE THREAD

**Issue by leniel59** [ihofmann/open-websoccer#108]
Hello I have translated all the files in the module folder into French I would like to know if there are other files to configure so that the new language is supported

**Comment by joseborges** [comment]
https://discord.gg/HCuGt7h

**Comment by joseborges** [comment]
@leniel59 do you already have FRENCH translation or do you need one?

**Comment by joseborges** [comment]
New link for anyone interested in openwebsoccer free development (hopefully this one doesn't expire)

https://discord.com/invite/6kWjdMY6Kx

**Comment by ihofmann** [comment]
Feel free to create a merge request if you translated everything.

**Comment by leniel59** [comment]
> [@leniel59](https://github.com/leniel59) do you already have FRENCH translation or do you need one?

Yes, it has been translated, but I have completely reworked and improved the project. I posted a few screenshots on Discord. It no longer looks anything like OpenWebSoccer, and I have made a lot of improvements. I will soon share a link with a demo account so that you can try it out for yourselves.
